scalyr_agent/builtin_monitors/mysql_monitor.py

```diff
@@ -96,6 +96,31 @@
     "``database_hostport``, but not both.",
     convert_to=six.text_type,
 )
+define_config_option(
+    __monitor__,
+    "use_ssl",
+    "Whether or not to use SSL when connecting to the MySQL server. Defaults to False.",
+    convert_to=bool,
+)
+define_config_option(
+    __monitor__,
+    "ca_file",
+    "Location of the ca file to use for the SSL connection. Defaults to None, which means the validity of the server "
```

Comment:
Perhaps should say ``server certificate verification won't be performed`` or similar since "validity of the server" sounds a bit weird.